Update K8 documentation to use AHA DNS name (SYN-8781) (#4178)
vertexproject/synapse: docs/synapse/devopsguide.rst

    for your environment.

  Aha naming
    In Kubernetes, we rely on the default naming behavior for services to find the Aha service via DNS. The Aha service
    uses a naming convention starting with ``aha`` followed by a two-digit number (e.g., ``aha00``) to support multiple instances.
    The service's DNS name is configured using the ``SYN_AHA_DNS_NAME`` environment variable, which follows the pattern
    ``<service-name>.<namespace>.svc.cluster.local``. For example, with the default namespace, the DNS name would be

>     ``aha00.default.svc.cluster.local``. The Aha service is configured to listen on ``0.0.0.0`` since we cannot bind the

Reviewer: ```suggestion
    ``aha00.default.svc.cluster.local``. The Aha service will automatically bind on ``0.0.0.0`` since we cannot bind the
```

    DNS label provided by Kubernetes prior to the Pod running Aha being available. The ``SYN_AHA_AHA_NETWORK`` value can be

>     set to any desired Certificate Authority name (e.g., ``dev.synapse``) and is used for SSL certificate validation.

Reviewer: ```suggestion
    set to any desired Certificate Authority name (e.g., ``dev.synapse``) and is used for SSL certificate validation between Synapse services.
```
